sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +21,4 @@
 [assembly: AssemblyTitle("Microsoft Azure SQL Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure SQL.")]
 [assembly: AssemblyVersion("3.0.0.0")]
```

Comment:
You should also bump the `AssemblyVersion`.

Reply:
From this guide: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec : 
AssemblyFileVersion (Must have)
AssemblyVersion: If it's a **major version** update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

Reply:
Yeah, that's optional. It's ok to leave it as it is.